app/helpers/mailer_helper.rb

```diff
@@ -0,0 +1,37 @@
+module MailerHelper
+  def poop_soon_subject(mfa_level)
```

Comment:
Phrasing! :sweat_smile: How about `mfa_required_soon_subject`? Same for other references in this file.

Reply:
lol yes! Sorry Aditya! Didn't realize you'd get to it before I could fixup my commit 🙈. I wanted to make sure the helper worked as expected first. Incoming fixup!